### PR TITLE
2021-03-22 summer talk submissions

### PR DESCRIPTION
**Pull request by FromAnkyra**
added a news article for summer term talks, changed the talks page to say that talks are open and also that talks take place on youtube (not in the pod)

### CODE REVIEW

File: regular/talks.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h3><em>How Robot Swarms Can Help Solve Hard Problems</em> by C Wringe</h3>
 In this talk, I will go over what a robot swarm is, and the sorts of problems they can solve. I will show what it means for a robot swarm to be scalable, flexible, and robust. Finally, I will go into some of the challenges faced by the field, and areas of current and future work.
 </p>
 <h3><em>An Introduction to Constraint Programming</em> by Jacob Allen</h3>
-<b>March 18th (Spring Week 10)</b>
+<b>March 18th (Spring Week 11)</b>
```
Comment (markspolakovs) [inline regular/talks.html:33]: You'll want to change the date here too no?
```suggestion
<b>March 25th (Spring Week 11)</b>
```

**Review by markspolakovs — COMMENTED** [review]
LGTM, modulo the comment

**Review by markspolakovs — APPROVED** [review]
Pulled locally and tested, all looks good to me.

Worth considering: should we move the "give a talk" noticebox at the bottom to the top? The talks page is getting quite long. Out of scope of this PR though.

**Comment by LukeMoll** [comment]
Looks fine functionally, I might make the wording around recording equipment stronger to encourage people who are unsure - ie "we can look into providing/we have funds to provide equipment to those who don't have suitable" rather than "we will see what we can do". Ex-committee and a suggestion, do reject if you disagree

**Comment by LukeMoll** [comment]
> Pulled locally and tested, all looks good to me.
> 
> Worth considering: should we move the "give a talk" noticebox at the bottom to the top? The talks page is getting quite long. Out of scope of this PR though.

See [#123](https://github.com/HackSoc/hacksoc.org/pull/123), in draft atm

**Comment by FromAnkyra** [comment]
@LukeMoll changing the wording feels like possibly a thing that should be added to the talks page revamp? (this is mainly because I don't have the brain to figure out words right now)

**Comment by LukeMoll** [comment]
Could do yeah. It doesn't need to block this being merged. I think I was
trying to keep content changes out of #123.

Happy to approve this (even though I wasn't asked 😛)

**Pull request merged**